plugins/scheme-tpm-enacttrust/main.go

```diff
@@ -101,6 +216,67 @@ func synthKeysFromParts(scope, tenantID string, parts *structpb.Struct) ([]strin
 	return []string{lookupKey.String()}, nil
 }
 
+func parseKey(keyString string) (*ecdsa.PublicKey, error) {
+	buf, err := base64.StdEncoding.DecodeString(keyString)
+	if err != nil {
+		return nil, err
+	}
+
+	key, err := x509.ParsePKIXPublicKey(buf)
+	ret, ok := key.(*ecdsa.PublicKey)
+	if !ok {
+		return nil, fmt.Errorf("could not extract EC public key; got [%T]: %v", key, err)
+	}
+
+	return ret, err
+}
+
+type Token struct {
```

Comment:
Could you please add a comment here explaining a bit about TPM token or a reference to some spec. especially I am not clear what the Raw here reflects to?

Reply:
This is not directly related to TPM or any other spec. This is merely an container internal to this implementation.